Doc/whatsnew/3.15.rst

```diff
@@ -155,3 +155,5 @@ Deprecated C APIs
 Removed C APIs
 --------------
 
+* :c:func:`!PyImport_ImportModuleNoBlock`: use :c:func:`PyImport_ImportModule`
+  instead.
```

Comment:
You may document that PyImport_ImportModuleNoBlock() is just an alias to PyImport_ImportModule() since Python 3.3. Same remark for the NEWS entry.